src/Kunstmaan/NodeBundle/EventListener/RenderContextListener.php

```diff
@@ -95,9 +95,13 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             //the SensioFrameworkExtraBundle kernel.view will handle everything else
             $event->setControllerResult((array) $parameters);
 
-            $template = new Template(array());
+            $template = new Template([]);
             $template->setTemplate($entity->getDefaultView());
 
+            $controllerBits = explode(':', $request->attributes->get('_controller'));
```

Comment:
This will only work for the old (and deprecated) `XXXBundle:Controller:action` notation. The new syntax `Namespace\For\Controller::method` doesn't work here

Reply:
@acrobat should I update this to only support the new syntax, or do we need to maintain backwards compatibility? The old syntax is still in use quite a bit internally at the moment

Reply:
I would indeed add a BC layer to support both the old and new syntax. This way we easily can remove the old deprecated notation once we update the supported symfony versions and are we able to fix this bug for older installations